sinch/core/clients/sinch_client_base.py

```diff
@@ -10,32 +9,27 @@
 class ClientBase(ABC):
     """
     Sinch abstract base class for concrete Sinch Client implementations.
-    By default this SDK provides two implementations - sync and async.
+    By default, this SDK provides two implementations - sync and async.
     Feel free to utilize any of them for you custom implementation.
     """
+    configuration = Configuration
+    authentication = AuthenticationBase
+    numbers = NumbersBase
+    conversation = ConversationBase
+    sms = SMSBase
```

Comment:
Don't you need to import and initialize `Verification` in this class too?
Also I've noticed that you are not using the `Base` pattern such as in `NumbersBase`, `ConversationBase` and `SMSBase` as for the async client, the async version for verification is imported as `import Verification as VerificationAsync`

Reply:
I've refactored this bit of code to comply with our unified naming convention form confluence.